hw/ip/aes/data/aes_testplan.hjson

```diff
@@ -130,9 +130,16 @@
             excercise the different reseeding configuations
             for reseeding every 8k blocks the DUT internal block counter will be manually changed to something close to 8k.
             to provoke the reseeding within reasonable simulation time '''
-      milestone: V2
+      milestone: V2S
       tests: ["aes_deinit"]
     }
+    {
+      name:fault_inject
+      desc: '''
+            Verify that injecting bit errors in one of the statemachines or the round counter triggers an error '''
```

Comment:
what is "round counter"? I don't see we test it in fi_vseq

Reply:
it will come in the next PR